modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -327,71 +405,34 @@ else if ((nestedFieldsDepth > 0) &&
 				_objectRelationshipLocalService.getObjectRelationships(
 					objectDefinition.getObjectDefinitionId());
 
-			Stream<ObjectRelationship> objectRelationshipsStream =
-				objectRelationships.stream();
-
-			List<ObjectRelationship> manyToManyRelationships =
-				objectRelationshipsStream.filter(
-					objectRelationship -> Objects.equals(
-						objectRelationship.getType(),
-						ObjectRelationshipConstants.TYPE_MANY_TO_MANY)
-				).collect(
-					Collectors.toList()
-				);
-
-			manyToManyRelationships.forEach(
+			objectRelationships.forEach(
 				objectRelationship -> {
-					try {
-						boolean reverse = objectRelationship.isReverse();
-
-						if (objectRelationship.isReverse()) {
-							objectRelationship =
-								_objectRelationshipLocalService.
-									fetchReverseObjectRelationship(
-										objectRelationship, false);
-						}
-
-						Pagination pagination = Pagination.of(1, 20);
-
-						List<com.liferay.object.model.ObjectEntry>
-							manyToManyRelatedObjectEntries =
-								_objectEntryLocalService.
-									getManyToManyRelatedObjectEntries(
-										objectEntry.getGroupId(),
-										objectRelationship.
-											getObjectRelationshipId(),
-										objectEntry.getObjectEntryId(), reverse,
-										pagination.getStartPosition(),
-										pagination.getEndPosition());
+					Object[] objectEntries = new ObjectEntry[0];
 
-						Stream<com.liferay.object.model.ObjectEntry>
-							manyToManyRelatedObjectEntriesStream =
-								manyToManyRelatedObjectEntries.stream();
+					if (Objects.equals(
+							objectRelationship.getType(),
+							ObjectRelationshipConstants.TYPE_MANY_TO_MANY)) {
 
-						map.put(
-							objectRelationship.getName(),
-							manyToManyRelatedObjectEntriesStream.map(
-								objectEntry1 -> {
-									try {
-										return _toDTO(
-											_getDTOConverterContext(
-												dtoConverterContext,
-												objectEntry1.
-													getObjectEntryId()),
-											nestedFieldsDepth - 1,
-											objectEntry1);
-									}
-									catch (Exception exception) {
-										exception.printStackTrace();
-
-										return null;
-									}
-								}
-							).toArray());
+						objectEntries = _getManyToManyRelationshipObjectEntries(
```

Comment:
Should we catch here also the PortalException as with OneToMany relationships?

Reply:
I moved the try-catch to the method like in the ManyToMany for consistency. Thanks